### PR TITLE
Fixed #905 (partially): Removed useless stuff from the nav

### LINKED ISSUE #905: Multiple design flaws
### Expected results
A complete redesign or just improvements to the design of the website, keeping in mind the principles of minimalism, clean design, consistency and blank spaces while preserving the actual content. This ideally should be achieved through partial fixes of this issue. 

Here are some resources for reference.
https://material.io/guidelines/
https://thenextweb.com/dd/2017/11/08/psychology-web-design/
https://envato.com/blog/scroll-hijacking/

### Actual results
Here are some (but not all) noticeable issues with the current design:
#### Consistency
The hover animations in the contents of the galleries aren't consistent.
In the mentors' gallery, the effect is a translucent overlay that has social links, whereas, in the Contributors' gallery, the overlay appears but the container moves down. Similarly, the Student Projects and Blogs gallery have totally different hover effects.
#### Smooth scroll
It'll be better to not have a smooth scroll, since it blocks users from normally scrolling down quickly.
More information can be found here: https://envato.com/blog/scroll-hijacking/
#### Navigation
On the navigation bar, only external links should be put. Too many links can be overwhelming to a casual visitor.
### Steps to reproduce the issue
Just open the site in a browser such as Google Chrome and you'll find the issues. If this is not the first time you're visiting the site, press F12, then right-click the Reload button and select the "Empty Cache and Hard Reload" option.
## Would you like to work on this
Yes, however, as mentioned before, we'll need multiple partial fixes. It'll be great if multiple people work on this.
For those interested in working on this, it's highly recommended that you have a look at each link mentioned above.

### PR DESCRIPTION
**Pull request by samyakbvs**
Removed all the internal links

[Deploy](https://samyakbvs.github.io/navabr-gci17.fossasia.org/)

### CODE REVIEW

**Review by aryan-02 — APPROVED** [review]

**Review by aryan-02 — APPROVED** [review]

File: _includes/header.html [hunk 1]
```diff
@@ -37,20 +37,8 @@
 					<div class="module left">
 						<ul class="menu">
 							<li>
-								<a href="{{ baseurl }}./#welcome">Welcome</a>
+								<a href="{{ baseurl }}./#welcome">Home</a>
 							</li>	                    
-							<li>
```
Comment (isuruAb) [inline _includes/header.html:42]: why we need to remove these things ?

Comment (samyakbvs) [reply _includes/header.html:42]: The issue maker asked me to do it

Comment (samyakbvs) [reply _includes/header.html:42]: The issue maker asked me to do the following changes

**Comment by samyakbvs** [comment]
@isuruAb

**Review by sayak119 — COMMENTED** [review]
Add Fixes #issue_number in PR description

**Comment by samyakbvs** [comment]
@sayak119 I can't bcs it will close the issue

**Comment by samyakbvs** [comment]
And there are other sub-issues in that issue which are not resolved

File: _includes/header.html (hunk 1 above)
Comment (harshit98) [reply _includes/header.html:42]: @samyakbvs Please elaborate. Even I'm not able to understand why you're removing these things?

**Comment by samyakbvs** [comment]
I am removing bcs the navbar was becoming overloaded

File: _includes/header.html [hunk 2]
```diff
@@ -37,20 +37,8 @@
...
-								<a href="{{ baseurl }}./#mentors">Mentors</a>
-							</li>
-							<li>
-								<a href="{{ baseurl }}./#contributors">Contributors</a>
-							</li>
```
Comment (abishekvashok) [inline _includes/header.html:47]: This is what our site needs. Should we remove it?

**Review by abishekvashok — CHANGES_REQUESTED** [review]

**Pull request closed without merging**